test/services/caServiceTest.js

```diff
@@ -0,0 +1,74 @@
+const createCaService = require('../../server/services/caService')
+
+describe('caService', () => {
+  let roService
+  let caService
+  const lduActiveClient = { isLduPresent: () => {} }
+
+  describe('Responsible Officer is allocated and Ldu determined', () => {
+    const responsibleOfficerOrError = {
+      isAllocated: true,
+      lduCode: 'ldu-123',
+    }
+
+    beforeEach(() => {
+      roService = {
+        findResponsibleOfficer: sinon.stub().resolves(responsibleOfficerOrError),
+      }
+    })
+
+    describe('getReasonForNotContinuing ', () => {
+      it('should return null because Ro is COM and ldu is active', async () => {
+        lduActiveClient.isLduPresent = sinon.stub().resolves(1)
```

Comment:
these should `isLduPresent ` be returning true or false.

Would be good to move  `caService = createCaService(roService, lduActiveClient)` into the `beforeEach`

Reply:
done
The only reason I duplicated them was, I thought it needed to come after the respective 
lduActiveClient.isLduPresent stubs, rather than before

Reply:
that's understandable but JS is 'pass by reference'. The lduActiveClient variable that you are manipulating is acting on the variable passed into the service (they both reference the same object in memory)